tests/v2/integration/steveapi/output.csv

```diff
@@ -1,55 +1 @@
 user,url,response
-user-a,https://localhost:8443/v1/secrets,[json/user-a_none_none.json](json/user-a_none_none.json)
```

Comment:
It looks like this file got emptied but the readme wasn't updated, is that intentional?

Reply:
Let me double check and rerun the tests, I might have forgotten to add it.

Reply:
You were right I forgot to remake the tables. It should be good now.